### ISSUE TITLE
Question regarding published artifacts

### ISSUE THREAD

**Issue by gintuoklis** [HodorNV/ALOps#72]
Hey Guys, 

I noticed that ALOps is publishing the same artifacts twice for the same build. Once in "Compile Log" folder, another time in "Dynamics 35 App"

![image](https://user-images.githubusercontent.com/16342128/73200286-c35e5480-4136-11ea-9192-5df8ada71d2f.png)

Is it intended behavior? 
It doubles then in the release pipelines as well.

![image](https://user-images.githubusercontent.com/16342128/73200611-6dd67780-4137-11ea-8541-8c0a05905472.png)

My code for Publish App has nothing special

```
- task: ALOpsAppPublish@1
  displayName: 'Publish AL-App'
  inputs:
    usedocker: true
    nav_artifact_app_filter: '*.app'   
    skip_verification: true

```

**Comment by AdminHodor** [comment]
Hello @gintuoklis ,

Good suggestions, does make sense. To avoid breaking changes on the Artifacts we changed to Log-Artifacts to the same "Dynamics 35 App" folder. From next release all artifacts will be located in the same root.

Kind regards

**Comment by AdminHodor** [comment]
Hello @gintuoklis ,
The changes for this are released (v1.427.1082), please check if this gives you the required result?
Kind regards,

**Comment by gintuoklis** [comment]
Thanks @AdminHodor, It's working fine now.
